pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -57,6 +57,7 @@ import (
 const (
 	floatingIPReleaseFinalizer = "kubermatic.io/release-openstack-floating-ip"
 	floatingIPIDAnnotationKey  = "kubermatic.io/release-openstack-floating-ip"
+	clientTimeout              = 3 * time.Minute
```

Comment:
Probably we should make it configurable.

Reply:
I wanted that as well but I'm not sure if I'd like to configure such a thing on the MD level :/ Maybe as a flag to machine-controller?!

Reply:
Personally I think 3 mins are also to much, personally I would prefer something around 1 minute with is already an increase by 4x.

Reply:
If we make it configurable this should also apply to all cloud provider api calls which is IMO a bigger change.

Reply:
1 minute should be more than enough, now that I think of it.

Reply:
I've switched this to 1 minute which should be enough. Having it configurable although would be good but it requires too much work since we'd then want something that is also configurable at KKP or KubeOne level. For now, I think setting it to 1 minute should be enough because imo 15 seconds is on the lower side when it comes to OpenStack.